hume/_batch/hume_batch_client.py

```diff
@@ -200,27 +202,37 @@ def _construct_request(
             request["notify"] = notify
         return request
 
-    def _submit_job_from_request(self, request_body: Any) -> BatchJob:
+    def _submit_job(
+        self,
+        request_body: Any,
+        files: Optional[List[Union[str, Path]]],
+    ) -> BatchJob:
         """Start a job for batch processing by passing a JSON request body.
 
         This request body should match the request body used by the batch API,
         including both the list of URLs and the models configuration.
 
         Args:
             request_body (Any): JSON request body to be passed to the batch API.
+            files: (List[Union[str, Path]]): JSON request body to be passed to the batch API.
```

Comment:
consider updating the documentation here

Reply:
Thanks! Good catch!